apps/test/unit/lib/ui/SchoolInfoInterstitialTest.jsx

```diff
@@ -70,6 +70,31 @@ describe('SchoolInfoInterstitial', () => {
     );
   });
 
+  it('closes the dialog when the dismiss button is clicked', () => {
+    const wrapper = shallow(
+      <SchoolInfoInterstitial
+        {...MINIMUM_PROPS}
+        scriptData={{
+          ...MINIMUM_PROPS.scriptData,
+          existingSchoolInfo: {}
+        }}
+      />
+    );
+    const wrapperInstance = wrapper.instance();
```

Comment:
@islemaster 
When onClick was set to this.dismissSchoolInfoForm, the error returned after the test was executed was:
```
 SchoolInfoInterstitial
      ✖ closes the school info interstitial modal when the dismiss button is clicked
        Chrome 75.0.3770 (Mac OS X 10.14.5)
      AssertionError: expected dismissSchoolInfoForm to have been called at least once, but it was never called

      + expected - actual

      
    at doAsserterAsyncAndAddThen (webpack:///node_modules/chai-as-promised/lib/chai-as-promised.js:289:21 <- test/entry-tests.js:42895:22)
    at Assertion.<anonymous> (webpack:///node_modules/chai-as-promised/lib/chai-as-promised.js:280:42 <- test/entry-tests.js:42885:47)
    at Assertion.get (webpack:///node_modules/chai/lib/chai/utils/overwriteProperty.js:50:0 <- test/entry-tests.js:50596:37)
    at Context.<anonymous> (webpack:///test/unit/lib/ui/SchoolInfoInterstitialTest.jsx:90:63 <- test/entry-tests.js:182161:84)

```